src/views/ForgotPasswordPageChange/index.js

```diff
@@ -1,31 +1,63 @@
+import AWS from 'aws-sdk';
 import React, { Component } from "react";
 import Form from "react-bootstrap/Form";
 import Button from "react-bootstrap/Button";
 import { Link, Route } from "react-router-dom";
 import LoginPage from "../LoginPage";
 
+const awsRegion = process.env.REACT_APP_AWS_REGION
 
+AWS.config.update({region:awsRegion});
+const cognitoidentityserviceprovider = new AWS.CognitoIdentityServiceProvider();
 
 class ForgotPasswordPageChange extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      code: undefined
+      password: undefined,
+      confirmpassword: undefined
     };
   }
 
   handleChange = event => {
-    const { info, value } = event.target;
-
-    this.setState({
-      [info]: value
-    });
-    console.log(this.state);
+    this.setState({[event.target.name]: event.target.value});
   };
 
   handleSubmit(event) {
     event.preventDefault();
-    console.log(this.state);
+
+    const { code } = this.props.location;
```

Comment:
There should be an error if a user attempts to access this page without first going through the whole forgot password process (which means there won't be a code in the state!) Look into doing the appropriate checks in the constructor or in another stage in the component lifecycle (https://reactjs.org/docs/react-component.html) to prevent the user from getting any further

Reply:
Added in this functionality. I couldn't do it in the constructor because none of the elements are rendered at that point, so I did the check as the mouse pointer enters the form area on the page. If it detects that the code is undefined, it disables the form and displays a message. Check it out!

Reply:
Awesome!